randovania/games/samus_returns/exporter/game_exporter.py

```diff
@@ -99,7 +101,7 @@ def _do_export_game(
 
             open_samus_returns_rando.patch_with_status_update(
                 export_params.input_path,
-                None,  # type: ignore
```

Comment:
Why does this have a `type: ignore`?

Reply:
This might be a left over from the previous pr

Reply:
Because I forgot to add `None` as type in the patcher. I changed it but we don't have it released.